idaes/surrogate/plotting/sm_plotter.py

```diff
@@ -60,64 +69,59 @@ def scatter2D(xdata, zdata, xtest, zfit, xlabels=None, zlabels=None, dz=None,
         zlabels = []
         for j in range(numouts):
             zlabels.append('z' + str(j+1))
-    if clabel is None:
-        clabel = 'Confidence Intervals'
 
-    pointidx = -1  # dummy index for each plot produced
-    fig, ax = [], []
+    if filename is not None:
+        if filename[-4:] != '.pdf':  # checking if extension is present
+            raise Exception('Filename does not end in .pdf, please amend '
```

Comment:
This is fine for now, but we should be using the "os" package to get the extension instead of relying on it being the last 4 characters.